chromadb/db/impl/sqlite.py

```diff
@@ -146,6 +155,8 @@ def reset_state(self) -> None:
                     """
             )
             for row in cur.fetchall():
+                if row[0].startswith("embedding_fulltext_search_"):
```

Comment:
This is needed due to a bug in Sqlite related to FTS5 tables:

- https://sqlite.org/forum/info/9292f41b2de7feb1
- https://github.com/mapnik/mapnik/issues/2483